<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Oscilloscope</class>
 <widget class="QWidget" name="Oscilloscope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>341</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QtCharts::QChartView" name="chartView" native="true"/>
      </item>
      <item>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="channelList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget { width: 4.8em }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="yRangeBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y Offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="yOffsetBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>-100000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="xRangeBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <item>
        <property name="text">
         <string notr="true">10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">5000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">10000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="holdReceiveBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hold Receive</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
